docs/user-guides/workspace-access/remote-desktops.md

```diff
@@ -47,6 +47,39 @@ Or use your favorite RDP client to connect to `localhost:3399`.
 
 The default username is `Administrator` and password is `coderRDP!`.
 
+### (Beta) Coder Desktop URI Handling
+
+[Coder Desktop](../desktop) can directly launch an RDP session without setting up port-forwarding using a URI handler.
+The URI format is:
+
+```text
+coder://<your Coder server name>/v0/open/ws/<workspace name>/agent/<agent name>/rdp?username=<username>&password=<password>
```

Comment:
is the password required here? I feel like I wouldn't want a plaintext password and would just expect to be prompted

Reply:
Kind of the whole point of this URI business is to make it "one click" to open, meaning we do want the password in the URI so we don't have to prompt.